test/polygon_test.cpp

```diff
@@ -3107,25 +3107,3 @@ TEST(Polygon, DISABLED_Woodgrain1) {
   });
   TestPoly(polys, 12, 0.00145);
 }
-
-TEST(Polygon, DISABLED_Woodgrain2) {
```

Comment:
I removed this one because upon inspection it's not actually geometrically valid - one of the inside contours in CCW instead of CW.